File: AndroidDev/testApp/app/src/main/java/com/celikel/testapp/MainActivity.java
package com.celikel.testapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        checkStatusOfApp();
    }

    private void checkStatusOfApp() {
        String targetAppName = "com.celikel.uninstalltracker";
        CheckApps.listInstalledApps(this);
        if (CheckApps.isAppInstall(this, targetAppName)) {
            Toast.makeText(MainActivity.this, (targetAppName + " found"), Toast.LENGTH_SHORT).show();
        } else {
            Toast.makeText(MainActivity.this, (targetAppName + " can not found"), Toast.LENGTH_SHORT).show();
        }
    }
}

File: 2Dice/app/src/main/java/com/celikel/learningandroid/MainActivity.kt
    package com.celikel.learningandroid

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.TextView
import android.widget.Toast

    class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val rollButton: Button = findViewById(R.id.button6)

        rollButton.setOnClickListener {
            //val toast = Toast.makeText(this, "Dice Rolled!", Toast.LENGTH_SHORT)
            //toast.show()
            //val resultTextView: TextView = findViewById(R.id.textView5)
            //resultTextView.text = "1"
            rollDice()
            rollDice2()
        }
    }

        private fun rollDice() {
            val dice = Dice(6)
            val diceRoll = dice.roll()
            val resultTextView: TextView = findViewById(R.id.textView5)
            resultTextView.text = diceRoll.toString()
        }
        private fun rollDice2() {
            val dice = Dice(6)
            val diceRoll = dice.roll()
            val resultTextView2: TextView = findViewById(R.id.textView6)
            resultTextView2.text = diceRoll.toString()
        }
    }

    class Dice(private val numSides: Int) {
        fun roll(): Int {
            return (1..numSides).random()
        }
    }